sagemaker_model_monitor/model_monitor_batch_transform/SageMaker-ModelMonitoring-Batch-Transform-Model-Quality-With-SageMaker-Pipelines-On-schedule.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>Please only use present tense (there will -&gt; there are, will be containing -&gt; contain). Make sure this applies to the rest of the notebook and other notebooks in this PR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3639/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
done

Reply:
<p>Maybe I should've been clearer: The manifests will contain -&gt; The manifests contain</p>

Reply:
Missed this one, updated

Reply:
<p>The manifests contains should be The manifests contain as it is plural</p>

Reply:
<p>Must address before merge!</p>